packages/wonder-blocks-form/package.json

```diff
@@ -4,6 +4,8 @@
   "design": "v1",
   "description": "Form components for Wonder Blocks.",
   "main": "dist/index.js",
+  "module": "dist/es/index.js",
+  "source": "index.js",
```

Comment:
Can you add a check to the pre-publish-check.js that confirms that module and source are set to the right values? (In addition to the `main` check that I have right now.)